src/lib/parsers/sol-json.c

```diff
@@ -282,6 +282,9 @@ sol_json_token_get_double(const struct sol_json_token *token, double *value)
     char *endptr;
     int r;
 
+    SOL_NULL_CHECK(token, -EINVAL);
```

Comment:
Actually this patch is invalid. Those parameters were already being checked with SOL_ATTR_NONNULL. I didn't realize that. thanks for pointing me that